learn/appendix_opengl/KHR_debug.md

```diff
@@ -0,0 +1,194 @@
+KHR_debug Extension
+===================
+`KHR_debug` is an extension that provides additional functions to the OpenGL API
+which aids development and debugging of OpenGL applications. This extension is
+available on many drivers and has been core since OpenGL version 4.3. Please
+read the previous article on the topic, [Debug Callback](debug_callback.md),
+before reading this as it provides better context into this extension.
+
+For versions of OpenGL later than 4.3 or if the extension is present you can
+directly use the function as defined in the `GL` class. For OpenGLES contexts
+use the `GL.Khr` class. The subclass version is a drop-in replacement for the
+core version for ES contexts. This extension only works when the context has
+the "Debug" flag set.
```

Comment:
For new OpenGL users this might be confused with C# debug mode, I think it's worth clarifying.